emsdk.py

```diff
@@ -1332,6 +1332,16 @@ def emscripten_npm_install(tool, directory):
       return False
 
   print('Done running: npm ci')
+
```

Comment:
Since the bootstrap process involves running `npm ci`, I wonder if this above running of `npm ci` is redundant, or if the addition use of the `--production` and `--no-optional` is somehow undone by running `bootstrap` here.

Reply:
I don't know, however, such a change of removing `npm ci` would be best for another commit for bisectability.

> if the addition use of the `--production` and `--no-optional` is somehow undone by running `bootstrap` here.

If that is the case, then there is something to fix in `bootstrap.py`, as currently Emscripten cannot be used without calling bootstrap.py.